charts/opentelemetry-operator/values.yaml

```diff
@@ -74,7 +74,7 @@ manager:
       repository: ""
       tag: ""
     # The Go instrumentation support in the operator is disabled by default.
-    # To enable it, use the operator.autoinstrumentation.go feature gate.
+    # To enable it, add --enable-go-instrumentation to extraArgs (source: https://github.com/open-telemetry/opentelemetry-operator/blob/5ddc44026eede78bd8cdf501e3689f9a143d9860/CHANGELOG.md?plain=1#L1153)
```

Comment:
We should update the chart to help set this feature.

Reply:
Good point @TylerHelmuth - do you suggest we have something specific in the `values.yaml` that, when enabled, adds the respective argument to `extraArgs`. Something like:

`go.autoinstrumentation.enabled = true` or reuse `operator.autoinstrumentation.go` but change the outcome to add ``--enable-go-instrumentation` to `extraArgs`

I'll then update accordingly.